### ISSUE TITLE
Doesn't run on Windows 8.1 profiles with special characters

### ISSUE THREAD

**Issue by citysushi** [steeve/plugin.video.pulsar#107]
I keep getting the "XBMC needs to restart" error when launching Pulsar, after numerous reinstalls. Same PC but other Windows 8.1 profile without "ë" character works fine.

http://pastebin.com/2B9CWqJT

Looks like a problem with special characters within %appdata% folder. Pulsar or Kodi bug? Other components work fine though.

Best regards.

**Comment by steeve** [comment]
Can you try, in `daemon.py` line 56-57 from:
```python
    binary_path = os.path.join(binary_dir, binary)
    dest_binary_path = os.path.join(dest_binary_dir, binary)
```

to

```python
    binary_path = os.path.join(binary_dir, binary).decode("utf-8")
    dest_binary_path = os.path.join(dest_binary_dir, binary).decode("utf-8")
```

**Comment by citysushi** [comment]
Thanks for the reply. I tried it, unfortunately no success.

This happens on Kodi startup, not having started the add-on yet:
http://pastebin.com/8CLEvbk4

**Comment by steeve** [comment]
Can you try a few lines earlier?
```
    binary_dir = os.path.join(ADDON.getAddonInfo("path"), "resources", "bin", "%(os)s_%(arch)s" % platform).decode("utf-8")
```

**Comment by steeve** [comment]
Remove the change I told you, and replace with:
```python
    binary_dir = binary_dir.decode("utf-8")
    dest_binary_dir = dest_binary_dir.decode("utf-8")

    binary_path = os.path.join(binary_dir, binary)
    dest_binary_path = os.path.join(dest_binary_dir, binary)
```

**Comment by citysushi** [comment]
Done. Still not working, though it's an other error:

http://pastebin.com/6uBeYii6

NOTICE: UnicodeEncodeError: 'ascii' codec can't encode character u'\xeb' in position 13: ordinal not in range(128)

**Comment by steeve** [comment]
Try this:
```python
    binary_dir = binary_dir.decode("latin1")
    dest_binary_dir = dest_binary_dir.decode("latin1")

    binary_path = os.path.join(binary_dir, binary)
    dest_binary_path = os.path.join(dest_binary_dir, binary)
```

**Comment by steeve** [comment]
Fixed in 6ed4d58f52737bec332348ac8f1fd44054039d40

You should be fine with v0.4.7
